src/mono/wasm/debugger/DebuggerTestSuite/InspectorClient.cs

```diff
@@ -98,6 +98,7 @@ public virtual Task<Result> SendCommand(SessionId sessionId, string method, JObj
 
             MessageId msgId = new MessageId(sessionId.sessionId, id);
             pending_cmds.AddOrUpdate(msgId, tcs,  (key, oldValue) => tcs);
+            logger.LogTrace($"SendCommand: to: {args}");
```

Comment:
Wrong file, I think you meant `src/mono/wasm/debugger/DebuggerTestSuite/FirefoxInspectorClient.cs`.